gulpfile.js

```diff
@@ -20,6 +20,7 @@ const qtcoreSources = [
   'src/engine/import.js',
   'src/engine/qml.js',
   'src/engine/*.js',
+  'src/qtbase/QObject.js',
```

Comment:
@ChALkeR @akreuzkamp I assumed this was a good idea to ensure that QObject.js is included before other files that may try to inherit from it (like QSizeF)? Just in case QObject turns into a class and has to be declared first.